scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/common/AverageAggregator.scala

```diff
@@ -25,9 +29,15 @@ import scala.collection.JavaConverters._
 /**
  * Return the average of all clients Tensors when calling aggregate
  */
-class AverageAggregator extends Aggregator[Table] {
+class AverageAggregator extends Aggregator {
```

Comment:
Do we actually need `AverageAggregator`? Why not rename it to `HflNNAggregator`?

Reply:
I used to consider FGBoost also use this average at the beginning. So far seems HFL is the only case to use averageAggregator? Please help me to confirm. @qiuxin2012 @qiyuangong